ui/components/app/transaction-decoding/components/decoding/address/address.container.js

```diff
@@ -0,0 +1,34 @@
+import { connect } from 'react-redux';
+import * as actions from '../../../../../../store/actions';
+import Address from './address.component';
+
+function mapStateToProps(state, ownProps) {
```

Comment:
Let's move this into the functional component, and get `recipientNickname` by selecting state using react-redux hooks

Reply:
Fixed.